cmake/cabana.patch

```diff
@@ -1,16 +1,6 @@
-https://github.com/ECP-copa/Cabana/pull/815
 --- CMakeLists.txt
 +++ CMakeLists.txt
-@@ -216,12 +216,6 @@
-   enable_testing()
+@@ -56,3 +56,2 @@ endforeach()
+ if(Kokkos_ENABLE_CUDA)
+-  kokkos_check(OPTIONS CUDA_LAMBDA)
```

Comment:
For my understanding, why do we have to disable this check in kokkos?

Reply:
The `kokkos_check()` function is only defined in the CMake-generated `KokkosConfigCommon.cmake` file. Including this file completely breaks the `FetchContent` build system.